frontend/src/displayMessages.js

```diff
@@ -1,23 +1,26 @@
 import React, { Component } from 'react';
 
 class DisplayMessages extends Component {
-    constructor(props) {
-      super(props);
-    }
+  constructor(props) {
+    super(props);
+  }
 
-    render () {
-        
+  render() {
+    return (
+      <div>
+        {this.props.messages.map(element => {
         return (
             <div>
-                <li>
-                    {this.props.messages.map(element => { 
-                        return <ul>{element.message}</ul>
-                    })}
-                </li>
+            <ul>
+                <li className="message">{element.message}</li>
+                <li className="fakeName">{element.fakeName}</li>
```

Comment:
Have we added css? I guess not, we should change fakeName to being bold or whatever.